src/wasm-interpreter.h

```diff
@@ -1879,6 +1879,9 @@ class ExpressionRunner : public OverriddenVisitor<SubType, Flow> {
     }
     return value;
   }
+  Flow visitExternConversion(ExternConversion* curr) {
+    WASM_UNREACHABLE("unimp");
+  }
```

Comment:
You also need to implement the other place here in this file that returns a nonconstant flow (search for the other `visitStringNew`).

Reply:
Why is this? None of the `ref.as_*` instructions are nonconstant. Is it just because these new instructions aren't implemented in the interpreter yet?

Reply:
Yes, an instruction must either be fully implemented, or return "nonconstant" (which makes the interpreter give up on it, but not fatally error).